protected/humhub/modules/admin/models/forms/PeopleSettingsForm.php

```diff
@@ -80,7 +83,7 @@ public function attributeLabels()
     public function attributeHints()
     {
         return [
-            'defaultSortingGroup' => Yii::t('AdminModule.user', 'Users of the selected group will be sorted first. "Last login" is used as additional sorting here for users inside the group and for other users and also if no group is selected.'),
+            'defaultSortingGroup' => Yii::t('AdminModule.user', 'Select a prioritised group whose members are displayed before all others when the sorting option \'Default\' is selected. The users within the group and the users outside the group are additionally sorted by their last login.'),
```

Comment:
What about `Selects the preferred group in which members appear before all other members when the sort option Default is selected. Users in and out of the group are further sorted by the last login.`?

Also just in case `prioritised` should be `prioritized`

Reply:
@Semir1212 What do you think?